engine/src/flutter/impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -88,12 +88,12 @@ struct TexImage2DData {
         type = GL_UNSIGNED_BYTE;
         break;
       case PixelFormat::kR32G32B32A32Float:
-        internal_format = GL_RGBA;
+        internal_format = GL_RGBA32F;
```

Comment:
Does some form of capability checking already exist for these?

Fixing the enum values here sgtm though.

Reply:
Err, doesn't look like it. I suppose it's GLES 3+ only.